DigitalOffice/Services/UserService/Repositories/GetUserInfoByIdRepository.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System;
+using System.Linq;
+using UserService.Database;
+using UserService.Database.Entities;
+using UserService.Mappers.Interfaces;
+using UserService.Repositories.Interfaces;
+
+namespace UserService.Repositories
+{
+    /// <summary>
+    /// Represents interface of repository. Provides method for getting user model from database.
+    /// </summary>
+    public class GetUserByIdRepository : IGetUserByIdRepository
```

Comment:
Rename UserRepository and IUserRepository
No need for individual repository for each query

Reply:
fixed